app/views/forms/sms_guide.html.erb

```diff
@@ -40,7 +40,7 @@
     </tr>
 
     <%# loop over each question %>
-    <% @form.smsable_questionings.each do |qing| %>
+    <% @form.smsable_questionings.each do |index, qing| %>
       <tr>
         <td class="rank"><%= qing.full_dotted_rank %>.</td>
```

Comment:
I think this should also be changed to `index` since decoding error messages will refer to the bare integer rank, not the full dotted one, and will confuse the heck out of everyone.